### PR TITLE
Gradle 6.9.2

### PR DESCRIPTION
**Pull request by mauricebarnum**
Descriptions of the changes in this PR:

### Motivation

Update gradle to 6.9.2 that has latest Log4Shell mitigations along with other bug fixes.

The mitigations aren't too compelling, but one less thing to question when scanning for vulnerabilities.

### Changes

1bd21a644 (HEAD -> gradle-6.9.2, msb-apache/gradle-6.9.2) Update gradle to 6.9.2
04ec49a3d Remove annoying println

### CODE REVIEW

**Comment by mauricebarnum** [comment]
rerun failure checks

**Review by nicoloboschi — APPROVED** [review]

**Review by Shoothzj — APPROVED** [review]

File: microbenchmarks/build.gradle [hunk 1]
```diff
@@ -41,12 +41,11 @@ jar {
...
     from {
         configurations.compileClasspath.collect {it.isDirectory() ? it : zipTree(it) }
     }
     exclude 'META-INF/*.RSA'
     exclude 'META-INF/*.SF'
     exclude 'META-INF/*.DSA'
     archiveBaseName = 'microbenchmarks'
-}
\ No newline at end of file
+}
```
Comment (Shoothzj) [inline microbenchmarks/build.gradle:51]: unnecessary change

**Review by merlimat — APPROVED** [review]
